app/src/main/java/org/jellyfin/androidtv/ui/startup/ManageServersScreen.kt

```diff
@@ -0,0 +1,29 @@
+package org.jellyfin.androidtv.ui.startup
+
+import androidx.core.os.bundleOf
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.auth.AuthenticationStore
+import org.jellyfin.androidtv.ui.preference.dsl.OptionsFragment
+import org.jellyfin.androidtv.ui.preference.dsl.optionsScreen
+import org.koin.android.ext.android.get
+
+class ManageServersScreen : OptionsFragment() {
+
+	override val screen by optionsScreen {
+		val authenticationStore = get<AuthenticationStore>()
+
+		val serverMap = authenticationStore.getServers()
+
+		setTitle(R.string.lbl_manage_servers)
+
+		for (entry in serverMap) {
+			link {
```

Comment:
```suggestion
		serverMap.forEach(entry -> link {
```

Also more readable to call it "server" instead of "entry".